src/assets/scripts/client/navigationLibrary/NavigationLibrary.js

```diff
@@ -317,6 +478,18 @@ class NavigationLibrary {
         return FixCollection.getFixRelativePosition(fixName);
     }
 
+    /**
+     *
+     *
+     * @for NavigationLibrary
+     * @method getProcedureLines
+     * @param procedureId {string}
+     * @return {array}
+     */
+     getProcedureLines(procedureId) {
```

Comment:
Showing a lint warning due to 9 spaces instead of 8 on this line only.

Reply:
fixed

Reply:
i still see this one overindented by one (the whole function is, actually. Only the docblock is correct. As of f130c56, that is.

Reply:
should be fixed now

Reply:
Yep! thanks